templates/democracy/contracts/DemocracyTemplate.sol

```diff
@@ -4,87 +4,90 @@ import "@aragon/templates-shared/contracts/BaseTemplate.sol";
 
 
 contract DemocracyTemplate is BaseTemplate {
+    bool constant private TOKEN_TRANSFERABLE = false;
```

Comment:
I think this should be true?

Reply:
Good catch! Will add some tests for that as well 👍